pyk/src/pyk/kore/syntax.py

```diff
@@ -2194,6 +2194,31 @@ def write(self, output: IO[str]) -> None:
             output.write('\n\n')
             module.write(output)
 
+    @cached_property
+    def axioms(self) -> tuple[Axiom, ...]:
+        return tuple(sent for module in self.modules for sent in module if isinstance(sent, Axiom))
+
+    def get_axiom_by_ordinal(self, ordinal: int) -> Axiom:
```

Comment:
I guess you're going to call this a lot, consider making the name shorter:

```python3
def get_axiom(self, ordinal: int) -> Axiom:
```

or

```python3
def get_axiom(self, *, ordinal: int) -> Axiom:
```

if you want to include `ordinal` in the call for unambiguity.

Reply:
Thanks for the advice! Indeed, it looks better! However, I would like to keep the same name we use in the LLVM Backend so that others can easily recognize it!